Unnecessary props removal, readme update and cleanup (#23)
Savedo/re-form: README.md

Configuration
=============

> `FormBuilder` component has props below. It should be a type of `FormBuilderPropsType<T>`. T refers to your object type having key (Field name, <string>) / values (Field options <FieldOptionsValueType>)

Reviewer: ```suggestion
The `FormBuilder` component has props as shown below. It should be typed as `FormBuilderPropsType<T>`, where T refers to your object type having key (Field name, <string>) / values (Field options <FieldOptionsValueType>)
```
